apps/service_profile/lib/profile/feed.ex

```diff
@@ -52,47 +53,42 @@ defmodule Profile.Feed do
     determine_reducers(dictionaries, reducers)
   end
 
-  defp find_bounding_box(reducers, dictionary, path) do
+  defp maybe_add_reducers(reducers, reducer, dictionary, path) do
+    unless includes_reducer?(reducers, reducer) do
+      reducers ++ create_reducers(reducer, dictionary, path)
+    else
+      reducers
+    end
```

Comment:
```suggestion
    reducers ++ create_reducers(reducer, dictionary, path)
```

This is all you need, isn't it? If a reducer exists, then `create_reducers` will return `[]`.

Reply:
I don't think, so with this change, 2 timestamps in the dictionary would result in 2 temporal_range reducers

Reply:
that sounds right to me.

Reply:
Yeah sorry you're right. I'm trying to read code in meetings and didn't fully work through that logic.